server/graphql/v2/object/OAuthAuthorization.js

```diff
@@ -38,6 +39,19 @@ export const OAuthAuthorization = new GraphQLObjectType({
       type: new GraphQLNonNull(GraphQLDateTime),
       description: 'The time of expiration',
     },
+    lastUsedAt: {
+      type: GraphQLDateTime,
+      description: 'The last time of token was used',
+      async resolve(authorization) {
+        const activity = await models.Activity.findOne({
```

Comment:
We should add a permission check here; only an admin of the token should be able to know its last usage date

Reply:
I added the permission check, can you check if the implementation is correct?

I also already updated GraphQL